tests/gcp/deployment/google_service_account/resource/service_accounts.yaml

```diff
@@ -0,0 +1,25 @@
+project_id: <project-id>
+components:
+  specs:
+    test-account:
+      account_id: test-account
+      display_name: test-account
+      description: Test account
+      disabled: false
```

Comment:
Both of these SAs have disabled as false, so not a great test. I'd be inclined to have a service account for each test and name it accordingly. Makes it clearer and more robust.